app/main/routes.py

```diff
@@ -92,106 +96,232 @@ def accessibility():
+    ayr_user = AYRUser(session.get("user_groups"))
+    if ayr_user.is_standard_user:
+        return redirect(
+            f"/browse/transferring_body/{ayr_user.transferring_body.BodyId}"
+        )
+    else:
+        # all access user (superuser)
+        for body in Body.query.all():
+            transferring_bodies.append(body.Name)
 
-    if transferring_body_id:
-        browse_type = "transferring_body"
-        browse_parameters["transferring_body_id"] = transferring_body_id
-        breadcrumb_values = {
-            0: {"transferring_body": Body.query.get(transferring_body_id).Name}
-        }
         filters = build_filters(request.args)
         sorting_orders = build_sorting_orders(request.args)
 
-    elif series_id:
-        browse_type = "series"
-        browse_parameters["series_id"] = series_id
-        series = Series.query.get(series_id)
-        body = series.body
-        breadcrumb_values = {
-            0: {"transferring_body_id": body.BodyId},
-            1: {"transferring_body": body.Name},
-            2: {"series": series.Name},
-        }
+        query = build_browse_all_query(
+            filters=filters,
+            sorting_orders=sorting_orders,
+        )
 
-        filters = build_filters(request.args)
-        sorting_orders = build_sorting_orders(request.args)
+        browse_results = query.paginate(page=page, per_page=per_page)
 
-    elif consignment_id:
-        browse_type = "consignment"
-        browse_parameters["consignment_id"] = consignment_id
-
-        consignment = Consignment.query.get(consignment_id)
-        body = consignment.series.body
-        series = consignment.series
-        breadcrumb_values = {
-            0: {"transferring_body_id": body.BodyId},
-            1: {"transferring_body": body.Name},
-            2: {"series_id": series.SeriesId},
-            3: {"series": series.Name},
-            4: {"consignment_reference": consignment.ConsignmentReference},
-        }
-
-        filters = build_browse_consignment_filters(request.args)
-        sorting_orders = build_sorting_orders(request.args)
+        total_records = db.session.query(
+            func.sum(query.subquery().c.records_held)
+        ).scalar()
+        if total_records:
+            num_records_found = total_records
+        else:
+            num_records_found = 0
+
+        return render_template(
+            "browse.html",
+            form=form,
+            current_page=page,
+            browse_type="browse",
+            results=browse_results,
+            transferring_bodies=transferring_bodies,
+            filters=filters,
+            sorting_orders=sorting_orders,
+            num_records_found=num_records_found,
+            dict_args={
+                k: v for k, v in request.args.items() if k not in "page"
+            },
+        )
+
+
+@bp.route("/browse/transferring_body/<uuid:_id>", methods=["GET"])
+@access_token_sign_in_required
+def browse_transferring_body(_id: uuid.UUID):
```

Comment:
why `'_id` and not just `'id`?

Reply:
id is a reserved keyword in Python and is conflicting when used

Reply:
Are you sure, I didn't think it was:
<img width="567" alt="Screenshot 2024-02-21 at 14 16 22" src="https://github.com/nationalarchives/da-ayr-beta-webapp/assets/42998618/e2157b23-1f95-48f8-8cba-bdd26ef6128d">

Reply:
as discussed on slack 'id' keyword shadow 

![image](https://github.com/nationalarchives/da-ayr-beta-webapp/assets/107114665/ba416c2d-8e82-4bb9-bee5-222754227c11)

Reply:
for others, id is a built in python function https://docs.python.org/3/library/functions.html.

I think due to its low usage in python code it is often renamed but making it distinct here avoids possible confusion 👌